app/assets/javascripts/tags.js.coffee

```diff
@@ -1,17 +1,34 @@
-$(document).on "click", ".switch_offer", (event) ->
-  $.ajax '/tags/offers',
-    type: 'GET'
-    dataType: 'html'
-    error: (jqXHR, textStatus, errorThrown) ->
-        $('.alpha_tag_list').html "AJAX Error: #{textStatus}"
+$ ->
+  $(".switch_offer-js").on "click", ->
```

Comment:
Yes, you're right, but in this case I considered that more elements like those (a switches for a very specific view) wouldn't be created programmatically :smile: